keyboards/dztech/dz65rgb/keymaps/adi_dz65rgb_ansi/readme.md

```diff
@@ -0,0 +1,13 @@
+# Adi's DZ65RGB Layout
+[Layout Visual](https://imgur.com/a/W7sH1mx)
+
+This layout modifies the extra row on the right and makes some of the key bindings easier to reach on Layer 1.
+## Step by Step
+1. Build your Firmware: run build command in directory
+<pre>* make dztech/dzrgb:adi_dz65rgb_ansi</pre>
```

Comment:
Use a [fenced code block](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#fenced-code-blocks) here instead of `<pre>` tags.